### PR TITLE
Follow up Site Icon: fixes

### PR DESCRIPTION
**Pull request by kebbet**
https://core.trac.wordpress.org/ticket/54370

This PR ddresses:
- buttons flow better on small viewports (taking account longer translations)
-`alt` strings are translateable
- removed unused ID's

Additional props to @afercia for commenting on original PR.

### CODE REVIEW

File: src/wp-admin/options-general.php [hunk 1]
```diff
@@ -125,14 +125,14 @@
 		<div class="favicon-preview">
 			<img src="<?php echo esc_url( admin_url( 'images/' . ( is_rtl() ? 'browser-rtl.png' : 'browser.png' ) ) ); ?>" class="browser-preview" width="182" alt="">
 			<div class="favicon">
-				<img src="<?php site_icon_url(); ?>" alt="Preview as a browser icon">
+				<img src="<?php site_icon_url(); ?>" alt="<?php _e( 'Preview as an app icon' ); ?>">
```
Comment (SergeyBiryukov) [inline src/wp-admin/options-general.php]:
```suggestion
				<img src="<?php site_icon_url(); ?>" alt="<?php esc_attr_e( 'Preview as an app icon' ); ?>">
```

File: src/wp-admin/options-general.php [hunk 2]
```diff
@@ -125,14 +125,14 @@
...
 			</div>
 			<span class="browser-title" aria-hidden="true"><?php echo get_bloginfo( 'name' ); ?></span>
 		</div>
-		<img class="app-icon-preview" src="<?php site_icon_url(); ?>" alt="Preview as an app icon">
+		<img class="app-icon-preview" src="<?php site_icon_url(); ?>" alt="<?php _e( 'Preview as an app icon' ); ?>">
```
Comment (SergeyBiryukov) [inline src/wp-admin/options-general.php]:
```suggestion
		<img class="app-icon-preview" src="<?php site_icon_url(); ?>" alt="<?php esc_attr_e( 'Preview as an app icon' ); ?>">
```

File: src/wp-admin/css/site-icon.css [hunk 3]
```diff
@@ -70,3 +69,10 @@
 	border-color: #b32d2e;
 	box-shadow: 0 0 0 1px #b32d2e;
 }
+
+.site-icon-section .action-buttons {
+	display: flex;
+	flex-wrap: wrap;
+	gap: 10px;
+}
+
```
Comment (mukeshpanchal27) [inline src/wp-admin/css/site-icon.css]: Empty line should be removed.

**Review by SergeyBiryukov — CHANGES_REQUESTED** [review]

File: src/wp-admin/options-general.php [hunk 4]
```diff
@@ -125,14 +125,14 @@
 		<div class="favicon-preview">
 			<img src="<?php echo esc_url( admin_url( 'images/' . ( is_rtl() ? 'browser-rtl.png' : 'browser.png' ) ) ); ?>" class="browser-preview" width="182" alt="">
 			<div class="favicon">
-				<img src="<?php site_icon_url(); ?>" alt="Preview as a browser icon">
+				<img src="<?php site_icon_url(); ?>" alt="<?php esc_attr_e( 'Preview as an app icon' ); ?>">
```
Comment (SergeyBiryukov) [inline src/wp-admin/options-general.php]:
```suggestion
				<img src="<?php site_icon_url(); ?>" alt="<?php esc_attr_e( 'Preview as a browser icon' ); ?>">
```
Not sure why the text was changed here 🙂

**Review by SergeyBiryukov — CHANGES_REQUESTED** [review]

File: src/wp-admin/options-general.php (hunk 4 above)
Comment (kebbet) [reply src/wp-admin/options-general.php]: Me copy pasting in a hurry..

File: src/wp-admin/css/site-icon.css (hunk 3 above)
Comment (kebbet) [reply src/wp-admin/css/site-icon.css]: Thanks for spotting this, [adf2331](https://github.com/WordPress/wordpress-develop/pull/6101/commits/adf2331ec1c396238cc0cf1550286c85378abe36) addresses this.

**Review by SergeyBiryukov — APPROVED** [review]

**Comment by SergeyBiryukov** [comment]
Thanks for the PR! Merged in [r57618](https://core.trac.wordpress.org/changeset/57618).

**Pull request closed without merging**